SPC Categorial Outlook Display.py

```diff
@@ -1,12 +1,18 @@
+import os
 import requests
 import matplotlib.pyplot as plt
 import matplotlib.patches as mpatches
-from matplotlib.offsetbox import OffsetImage, AnnotationBbox
 import geopandas as gpd
+from matplotlib.offsetbox import OffsetImage, AnnotationBbox
 
-# Function to fetch the SPC outlook from the GeoJSON URL
+# Function to create the output directory
+def create_output_directory():
+    current_directory = os.path.dirname(os.path.abspath(__file__))
+    output_directory = os.path.join(current_directory, 'output')
+    os.makedirs(output_directory, exist_ok=True)
+    return output_directory

```

Comment:
New Function to detect and create an output directory for the automated outputted imaged to be placed